MarsRover.Tests/OutputFormatterShould.cs

```diff
@@ -9,10 +9,10 @@ public class OutputFormatterShould
     {
         [Theory]
         [MemberData(nameof(OutputFormatterShouldTestData.FormatMapNoRoverTestData), MemberType = typeof(OutputFormatterShouldTestData))]
-        public void FormatMap_ReturnsMapWithNoRover_GivenMap(List<Square> squares, int width, int height, params string[] inputs)
+        public void FormatMap_ReturnsMapWithNoRover_GivenMap(List<Square> squares, int width, int height, params string[] lines)

```

Comment:
The function name is `DisplayMap` rather than `FormatMap`. Perhaps it should be `FormatMap`